hono/auth/jwt.ts

```diff
@@ -11,6 +11,9 @@ import { apiLogger } from '../logger';
 const secret = process.env.JWT_SECRET!;
 const webmasters = process.env.WEBMASTERS!.split(',');
 
+export const academicYear =
+  new Date().getFullYear() - Math.floor(new Date().getFullYear() / 100) * 100;
```

Comment:
minor thing but if someone is using Mad in the  new year before sept 26 this would show 26. 
```suggestion
const now = new Date();
const year = now.getFullYear();
const month = now.getMonth() + 1; 
const academicYearStart = month >= 10 ? year : year - 1;
const academicYear = academicYearStart % 100; 
//remove magic number nonsense (could also make the check for September rather than October)
```

Reply:
Good catch, ty